SkyFloatingLabelTextField/SkyFloatingLabelTextFieldExample/Example0/ShowcaseExampleViewController.swift

```diff
@@ -213,10 +210,12 @@ class ShowcaseExampleViewController: UIViewController, UITextFieldDelegate {
         }
     }
     
-    func isValidEmail(str:String?) -> Bool {
-        let emailRegEx = "^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$"
+    // MARK: - validation
+
+    // NOTE: source: http://emailregex.com
+    func validateEmail(_ candidate: String) -> Bool {
```

Comment:
I'd actually prefer if we changed the email validation to only look for some characters and an @ sign.

Something like `.+@.+`. Correctly validating emails using regexes is notoriously difficult and I'd prefer if we didn't contribute to the idea that validating emails addresses with regex is a good idea.